core/src/jvmMain/kotlin/io/kinference/compiler/generation/operators/OperatorGenerator.kt

```diff
@@ -0,0 +1,22 @@
+package io.kinference.compiler.generation.operators
+
+import com.squareup.kotlinpoet.CodeBlock
+import io.kinference.compiler.generation.operators.math.AddGenerator
+import io.kinference.compiler.generation.operators.math.MulGenerator
+import io.kinference.operators.Operator
+import io.kinference.operators.math.Add
+import io.kinference.operators.math.Mul
+
+class OperatorGenerator(
```

Comment:
Should we add some comment somewhere that we should add a new operator into this generator? Maybe to the base class